Tasks/KubernetesV1/src/kubernetescommand.ts

```diff
@@ -42,19 +42,40 @@ function getCommandOutputFormat(kubecommand: string) : string[] {
     return args;
 }
 
-function getCommandConfigurationFile() : string[] {
-    var args: string[] =[];
-    var useConfigurationFile : boolean  =  tl.getBoolInput("useConfigurationFile", false);
-    if(useConfigurationFile) {
-        var configurationPath = tl.getInput("configuration", true);
-        if(configurationPath && tl.exist(configurationPath))
-        {
+function getCommandConfigurationFile(): string[] {
+    var args: string[] = [];
+    var useConfigurationFile: boolean = tl.getBoolInput("useConfigurationFile", false);
+    let configurationPath = tl.getPathInput("configuration", false);
+    var inlineConfiguration = tl.getInput("inline", false);
+
+    if (!tl.filePathSupplied("configurationPath")) {
+        configurationPath = null;
+    }
+
```

Comment:
Instead of setting it to `null`
you could have a single var like 
```js
let isInvalidConfig = !!configurationPath == !!tl.filePathSupplied("configurationPath");
```

`!!` would convert it to `true` or `false`